tests/test.web-ext.js

```diff
@@ -17,15 +17,14 @@ describe('web-ext', () => {
   // This is a smoke test just to make sure the command line script
   // doesn't explode.
 
-  it('prints the current version number', (done) => {
-    let pkg = JSON.parse(
-      readFileSync(path.join(__dirname, '..', '..', 'package.json')));
-    let cmd = 'bin/web-ext --version';
-    shell.exec(cmd, {silent: true}, (code, output) => {
-      assert.equal(output.trim(), pkg.version);
-      assert.equal(code, 0, output);
-      done();
-    });
+  it('shows help', (done) => {
+    let webExt = path.join(path.resolve(__dirname), '..', 'bin', 'web-ext');
+    shell.exec(`${webExt} --help`, {silent: true},
```

Comment:
I'm tempted to kill this test because it's slow and sometimes causes a timeout on Travis. It seems useful though because it will cover a catastrophic error in the `bin/web-ext` file and maybe also `program.main()` (although that has some coverage).